configure.ac

```diff
@@ -5319,7 +5319,37 @@ AC_DEFINE(MONO_INSIDE_RUNTIME,1, [Disable banned functions from being used by th
 mono_build_root=`pwd`
 AC_SUBST(mono_build_root)
 
-mono_runtime=mono/mini/mono
+AC_ARG_WITH(msvc, [  --with-msvc=yes,no      Build with Visual Studio (msvc) (defaults to no)],[],[with_msvc=no])
```

Comment:
Use `AC_ARG_ENABLE` so you have `--enable-msvc`. We overuse the `--with-*` which is not originally meant for a yes/no parameter. Then propagate that `ENABLE_MSVC` variable instead of `WITH_MSVC`.

Reply:
OK, will change. Original used --enable-msvc at a start but then I looked for other patterns that seemed to be using --with-*.